modules/network-observability-netobserv-cli-install.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+
+// * observability/network_observability/netobserv_cli/netobserv-cli-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-cli-install_{context}"]
+= Installing the Network Observability CLI
+Installing the Network Observability CLI (`oc netobserv`) is a separate procedure from the Network Observability Operator installation. This means that, even if you have the Operator installed from OperatorHub, you need to install the CLI separately. 
+
+[NOTE]
+====
+You can optionally use Krew to install the `netobserv` CLI plugin. For more information, see "Installing a CLI plugin with Krew". 
+====
+
+.Prerequisites
+* You must install the OpenShift CLI (`oc`). 
+* You must have a macOS or Linux operating system.
+
+.Procedure
+//check link accuracy before merge
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/netobserv/latest/[netobserv cli tar file].
+. Unpack the archive:
++
+[source,terminal]
+----
+$ tar xvf netobserv-cli.tar.gz
+----
+. Make the file executable:
++
+[source,terminal]
+----
+$ chmod +x ./build/oc-netobserv
+----
+. Move the extracted `netobserv-cli` binary to a directory that is on your `PATH`, such as `/usr/local/bin/.`:
++
+[source,terminal]
+----
+$ sudo mv ./build/oc-netobserv /usr/local/bin/
+----
+
+.Verification
+
+* After you install the netobserv CLI, verify that it is available:
```

Comment:
```suggestion
* After you install the `netobserv` CLI, verify that it is available:
```

Reply:
I'm just going to say Network Observability CLI here because otherwise, I will need to call it `oc netobserv`.